### PR TITLE
adds icons to tags

### PR DESCRIPTION
**Pull request by NoahThomlison**

### CODE REVIEW

**Comment by JakeHildy** [comment]
@NoahThomlison can you run `npm install` so we get the package lock updated too?

File: src/tag/Tag.js [hunk 1]
```diff
@@ -35,7 +85,9 @@ const Tag = (props) => {
       onClick={onClick}
       data-testid={dataTestId}
     >
+      {startIcon && <Icon icon={startIcon} size={size} />}
       <StyledTagLabel>{children}</StyledTagLabel>
+      {endIcon && <Icon icon={endIcon} size={size} />}
```
Comment (JakeHildy) [inline src/tag/Tag.js:90]: Do we need startIcon or endIcon here? If we have either startIcon or endIcon the first block will render

**Review by JakeHildy — CHANGES_REQUESTED** [review]

File: src/tag/Tag.js (hunk 1 above)
Comment (NoahThomlison) [reply src/tag/Tag.js]: Really I dont need the second block at all. Everything should work with the first block and I should remove the conditional wrapping it. At one point when I was styling I had thought I needed two different returns to change the wrapper around the Tag. Didnt end up needing that but missed removing that extra code I put in.

File: src/tag/Tag.js [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
+  const {
+    children,
+    className,
+    onClick,
+    onDelete,
+    variant,
+    dataTestId,
+    startIcon,
+    endIcon,
+    size
```
Comment (JakeHildy) [inline src/tag/Tag.js:21]: I talked with Andrea and she thinks this size isn't going to change ever so I kinda think we should just remove it. Thoughts?

Comment (NoahThomlison) [reply src/tag/Tag.js:21]: Yep no problemo.

**Review by JakeHildy — APPROVED** [review]

**Pull request merged**